.github/workflows/go-cross.yml

```
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        go-version: [ 1.18, 1.19, 1.x ]
+        go-version: [ 1.19, '1.20', 1.x ]
```

Let's generally quote things that could be mistaken as floating point numbers by YAML parsers:

```suggestion
        go-version: [ '1.19', '1.20', 1.x ]
```